code/workspaces/web-app/src/api/projectsService.js

```diff
@@ -25,4 +25,26 @@ function loadProjectInfo(projectKey) {
     .then(errorHandler('data.project'));
 }
 
-export default { loadProjects, loadProjectInfo };
+function createProject(project) {
+  const mutation = `
+    CreateProject($project: ProjectCreationRequest) {
+      createProject(project: $project) {
+        id, key, name, description, collaborationLink, accessible
```

Comment:
I'm not sure you need to return all of the details here. As far as I can tell you don't care about the returned object, just that the request is successful.

Reply:
Update to just return `id`. Have to extract something from the return value otherwise fails with a `400`.